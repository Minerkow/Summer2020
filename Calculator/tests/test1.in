a = ?;
v  = 957 /895 -525 -337 -229 /517 /454 *941 *a ;
m  = 213 /(((721 -((498 ))/((487 )*(384 ))*923 )-978 +452 -514 /371 /800 -186 )-466 /975 -467 -892 /834 )-581 +620 ;
p  = 51 /(269 +235 /(619 -(420 )));
print v;
print m;
print p;