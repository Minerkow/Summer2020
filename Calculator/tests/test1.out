-861
39
0